frontend/src/lib/components/learningkit/displays/UserItem.svelte

```diff
@@ -61,50 +70,77 @@
 	}
 </script>
 
-<div class="card preset-filled-surface-100-900 flex w-full items-center justify-between p-4">
+<div
+	class="card preset-filled-surface-100-900 flex w-full items-center justify-between p-4 {showProgress &&
+	isCompleted
+		? 'border-l-4 border-green-500'
+		: 'border-surface-100-900 border-l-4'}"
+>
 	<div class="flex w-full max-w-sm items-center gap-4 truncate">
-		<Avatar name="{user.surname} {user.name}" classes="h-10 w-10" />
+		<div class="relative">
+			<Avatar name="{user.surname} {user.name}" classes="h-10 w-10" />
+			{#if showProgress && isCompleted}
+				<CheckCircle2
+					class="bg-surface-100-900 absolute -top-1 -right-1 h-4 w-4 rounded-full text-green-500"
+				/>
+			{/if}
+		</div>
 		<div class="flex flex-col">
 			<p class="truncate font-bold">{user.surname} {user.name}</p>
 			<p class="truncate text-sm text-gray-500">{user.username}</p>
 		</div>
 	</div>
-	<div class="flex gap-2">
-		{#if isUsersView}
-			<button
-				type="button"
-				class="btn preset-outlined-surface-500"
-				onclick={() => goto(`/users/${user.uuid}/edit-form`)}
-			>
-				{$_('common.edit')}
-			</button>
+
+	{#if showProgress && progressPercentage !== undefined}
+		<div class="flex items-center space-x-4">
+			<div class="w-24 text-right">
+				<Progress
+					value={progressPercentage}
+					max={100}
+					height="h-2"
+					meterBg={isCompleted ? 'bg-green-500' : 'bg-primary-500'}
+				/>
+				<span class="text-surface-500-400 text-xs">{progressPercentage}%</span>
+			</div>
+		</div>
+	{:else}
+		<div class="flex gap-2">
+			{#if isUsersView}
+				<button
+					type="button"
+					class="btn preset-outlined-surface-500"
+					onclick={() => goto(`/users/${user.uuid}/edit-form`)}
```

Comment:
Looks like this should be a link instead of a button. use a with btn classes and href instead of goto